modules/csharp/examples/code_snippets/Program.cs

```diff
@@ -2455,80 +2452,105 @@ public Document Resolve(Conflict conflict)
     // end::merge-conflict-resolver[]
  }
 
-#warning p2p-act-rep-func used, but contains nothing
 // tag::p2p-act-rep-func[]
+// . . . preceding code. for example . . .
+    private static ListenerToken _thisListenerToken;
+    var Database thisDB;
+    // . . . other code . . .
 
-#warning p2p-act-rep-config-type used, but contains nothing
 // tag::p2p-act-rep-config-type[]
+// Set replicator type
+thisConfig.ReplicatorType = ReplicatorType.PushAndPull;
 
 // end::p2p-act-rep-config-type[]
 
-#warning autopurge-override used, but contains nothing
 // tag::autopurge-override[]
 // Set autopurge option
 // here we override its default
+thisConfig.EnableAutoPurge = false; // <.>
 
 // end::autopurge-override[]
 
-#warning p2p-act-rep-config-cont used, but contains nothing
 // tag::p2p-act-rep-config-cont[]
 // Configure Sync Mode
+thisConfig.Continuous = true; // default value
 
 // end::p2p-act-rep-config-cont[]
 
-#warning p2p-act-rep-config-self-cert used, but contains nothing
 // tag::p2p-act-rep-config-self-cert[]
 // Configure Server Security -- only accept self-signed certs
+thisConfig.AcceptOnlySelfSignedServerCertificate = true; // <.>
 
 // end::p2p-act-rep-config-self-cert[]
-
 // Configure Client Security // <.>
 
-#warning p2p-act-rep-auth used, but contains nothing
 // tag::p2p-act-rep-auth[]
 // Configure basic auth using user credentials
+    thisConfig.Authenticator = new BasicAuthenticator("Our Username", "Our Password");
 
 // end::p2p-act-rep-auth[]
 
-#warning p2p-act-rep-start-full used, but contains nothing
 // tag::p2p-act-rep-start-full[]
 // Initialize and start a replicator
 // Initialize replicator with configuration data
+var thisReplicator = new Replicator(thisConfig); // <.>
 
-#warning p2p-act-rep-add-change-listener used, but contains nothing
 // tag::p2p-act-rep-add-change-listener[]
-#warning p2p-act-rep-add-change-listener-label used, but contains nothing
 // tag::p2p-act-rep-add-change-listener-label[]
 //Optionally add a change listener // <.>
 // end::p2p-act-rep-add-change-listener-label[]
+_thisListenerToken =
+      thisReplicator.AddChangeListener((sender, args) =>
+        {
+          if (args.Status.Activity == ReplicatorActivityLevel.Stopped) {
+              Console.WriteLine("Replication stopped");
+          }
+        });
 
 // end::p2p-act-rep-add-change-listener[]
 
-#warning p2p-act-rep-start used, but contains nothing
 // tag::p2p-act-rep-start[]
 // Start replicator
+ thisReplicator.Start(); // <.>
```

Comment:
This is the same as starting any other replicator...does it really need a separate snippet?